sim.py

```diff
@@ -184,10 +184,11 @@ def buildSim(self, ldr):
                 # If an object entry in placed_objs does not
                 # have a position, it is ignored.
                 if "x" in o and "y" in o:
-                    newobj = ldr.copyObjTemplate(oid)
+                    newobj = ldr.copyObjTemplate(oid)  # Point C
                     data = o
                     data["uuid"] = uuid.uuid4()
-                    newobj.place(data)
+                    if newobj.place:
```

Comment:
What is the significance of adding the if statement? Should it always result to True? If so, this could be a good spot to add error logging if it is ever false.

Reply:
I do not remember the purpose of the if statement. I am not sure what to do about it. Would be interested in discussing?